app/Services/Course/StoreCourseService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace app\Services\Course;
```

Comment:
@KosukeChimata 
次のように変更する必要があるので、私の方で修正しますね！
以降は`namespace`の命名も確認いただくようにお願いします！

```suggestion
namespace App\Services\Course;
```

Reply:
誠に申し訳ございませんでした。
ご訂正ありがとうございます。